samples/managePower.py

```diff
@@ -0,0 +1,25 @@
+import argparse
+import vm_automation
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-k", "--keyword", help="VM search parameter")
+    parser.add_argument("-p1", "--powerOn", help="power on vm before snapshot", action="store_true")
```

Comment:
Will change help text on this and next to "power on" && "power off" as I land, since this is "managePower.py"